recipe/meta.yaml

```diff
@@ -1,33 +1,36 @@
 {% set name = "popt" %}
-{% set version = "1.16" %}
+{% set version = "1.19" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: ftp://anduin.linuxfromscratch.org/BLFS/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: e728ed296fe9f069a0e005003c3d6b2dde3d9cad453422a10d6558616d304cc8
+  - url: https://ftp.osuosl.org/pub/rpm/{{ name }}/releases/{{ name }}-1.x/{{ name }}-{{ version }}.tar.gz
```

Comment:
The dev_url https://github.com/rpm-software-management/popt points to http://ftp.rpm.org/popt/. 
Should the source url be taken from http://ftp.rpm.org/popt/releases/popt-1.x/?
https://osuosl.org/ is an URL of Oregon State University Open Source Lab.

How can we verify that http://ftp.rpm.org is identical to OSUOSL? It looks like a mess.

Reply:
That's the url referenced in the release note at least: https://github.com/rpm-software-management/popt/releases

_[popt 1.19](https://github.com/rpm-software-management/popt/releases/tag/popt-1.19-release) [Latest](https://github.com/rpm-software-management/popt/releases/latest)
Download
https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
SHA256SUM: c25a4838fc8e4c1c8aacb8bd620edb3084a3d63bf8987fdad3ca2758c63240f9_

Reply:
I updated to ftp.rpm.org in the end. Thanks!